### PR TITLE
Doc Cleanups and Adding Full Storage Node and Consensus Full Nodes

### PR DESCRIPTION
**Pull request by YazzyYaz**
- General Doc Cleanup
- Create Full Storage Node doc
- Create Consensus Full Node doc
- Add Sections for Consensus and Data Availability in "Run A Node"
- Move SystemD to it's own page and link nodes to it to run

### CODE REVIEW

File: docs/nodes/consensus-full-node.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+If the file was created successfully you will be able to see its content:
+
+```sh
+cat /etc/systemd/system/celestia-appd.service
+```
+
+You must now set a few configs in your `config.toml`. Learn more about the
+config.toml file [here](../nodes/config-toml.md)
+
+Run the following command to set timeout to 25 seconds:
```
Comment (liamsi) [inline docs/nodes/consensus-full-node.md:129]:
```suggestion
Run the following command to set the commit timeout to 25 seconds:
```

Comment (liamsi) [reply docs/nodes/consensus-full-node.md:129]: I'm actually concerned that we communicated already different things to validators here. Originally, we said 5s, then 25s, then 30s, here again 25s 🤔

Comment (mindstyle85) [reply docs/nodes/consensus-full-node.md:129]: we said 30s block times but that setting was only communicated to be 25s

File: docs/nodes/consensus-full-node.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+```sh
+sudo systemctl enable celestia-appd
+sudo systemctl start celestia-appd
+```
+
+Check if daemon has been started correctly:
+
+```sh
+sudo systemctl status celestia-appd
```
Comment (adlerjohn) [inline docs/nodes/consensus-full-node.md:152]:
```suggestion
systemctl status celestia-appd
```

File: docs/nodes/consensus-full-node.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+```
+
+Check daemon logs in real time:
+
+```sh
+sudo journalctl -u celestia-appd.service -f
```
Comment (adlerjohn) [inline docs/nodes/consensus-full-node.md:158]:
```suggestion
journalctl -u celestia-appd.service -f
```

File: docs/nodes/consensus-full-node.md (hunk 2 above)
Comment (mindstyle85) [reply docs/nodes/consensus-full-node.md:152]: i dont like this change as it suggests people run software with root user which isnt good practice

Comment (adlerjohn) [reply docs/nodes/consensus-full-node.md:152]: Can you clarify?

Comment (mindstyle85) [reply docs/nodes/consensus-full-node.md:152]: my mistake, i thought i saw you did the same for `systemctl start celestia-appd` which doesnt work, but those two work yes

**Comment by YazzyYaz** [comment]
@adlerjohn @liamsi added your commits to the PR

**Review by adlerjohn — COMMENTED** [review]
lint fail

**Review by adlerjohn — APPROVED** [review]
utACK

**Comment by YazzyYaz** [comment]
Closes https://github.com/celestiaorg/docs/issues/62

**Pull request merged**